src/lib.rs

```diff
@@ -499,6 +519,7 @@ impl TOTP {
 
     /// Parse the TOTP parts from the standard otpauth URL
     #[cfg(feature = "otpauth")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "otpauth")))]
```

Comment:
On a non-pub method?

Reply:
Didn't notice, sorry

Reply:
No problem, if you can just fix that I'll merge and figure out the few things before publishing :)